### PR TITLE
optimisation runs recover default

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

File: apps/remix-ide-e2e/src/tests/compiler_api.test.ts [hunk 1]
```diff
@@ -111,11 +119,28 @@ const jsCompileWithOptimization = `(async () => {
...
+})()`
+
+const jsCompileWithOptimizationDefault = `(async () => {
+  try {
+      const contract = {
+          "storage.sol": {content : \`${simpleContract}\` }
+      }
+      console.log('compile')
+      const params = {
+          optimize: false,
```
Comment (yann300) [inline apps/remix-ide-e2e/src/tests/compiler_api.test.ts:134]: don't you need to specify compiler version, evm version, language, ... ?

Comment (LianaHus) [reply apps/remix-ide-e2e/src/tests/compiler_api.test.ts:134]: I'm changing only this, everything else can go with default

Comment (yann300) [reply apps/remix-ide-e2e/src/tests/compiler_api.test.ts:134]: ok I don't know if default are handled for these params. Don't they need to be specified?

File: apps/remix-ide-e2e/src/tests/compiler_api.test.ts [hunk 2]
```diff
@@ -34,6 +34,14 @@ module.exports = {
     .journalChildIncludes(`\\"optimizer\\":{\\"enabled\\":true,\\"runs\\":300}`)
   },
 
+  'Should compile using "compileWithParamaters" API with optimization off check default runs': function (browser: NightwatchBrowser) {
+    browser
+    .addFile('test_jsCompileWithOptimizationDefault.js', { content: jsCompileWithOptimizationDefault })
```
Comment (cleytontiagosorinho) [inline apps/remix-ide-e2e/src/tests/compiler_api.test.ts:39]: Ola bom dia

**Review by cleytontiagosorinho — APPROVED** [review]

File: apps/remix-ide-e2e/src/tests/compiler_api.test.ts (hunk 2 above)
Comment (cleytontiagosorinho) [reply apps/remix-ide-e2e/src/tests/compiler_api.test.ts:39]: Os melhores da lista

**Review by cleytontiagosorinho — APPROVED** [review]
E isso mesmo

**Comment by cleytontiagosorinho** [comment]
E tudo que eu quero os melhores rendimentos obrigado

**Review by cleytontiagosorinho — APPROVED** [review]

**Review by cleytontiagosorinho — APPROVED** [review]

**Review by yann300 — APPROVED** [review]

**Pull request merged**